libs/base/sim/control.ts

```diff
@@ -160,14 +160,12 @@ namespace pxsim.control {
     }
 
     export function __log(priority: number, str: string) {
-        let prefix = "";
         switch (priority) {
-            case 0: prefix = "d>"; break;
-            case 1: prefix = "l>"; break;
-            case 2: prefix = "w>"; break;
-            case 3: prefix = "e>"; break;
+            case 0: console.debug("d>" + str); break;
+            case 1: console.log("l>" + str); break;
+            case 2: console.warn("w>" + str); break;
+            case 3: console.error("e>" + str); break;
```

Comment:
@pelikhan any reason this would be problematic?

Reply:
@riknoll @jwunderl @shakao  @abchatra - any reason sending messages to console not always with console.log but with console.warn etc would be problematic? it will give better experience when looking at JS console when running sim

Reply:
Seems fine to me~

Reply:
looks good to me.